Bowling/BowlingGame.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Bowling.Exceptions;
+using Bowling.UtilityComponents;
+
+namespace Game
+{
+    public class BowlingGame
+    {
+        private List<Player> players = new List<Player>();
+        private List<Frame> framesScoreBoard = new List<Frame>();
+        private int CurrentFrameIndex = 0;
+        private int bonus = -1;
+        public int NumberOfPlayers { get; set; }
```

Comment:
I would make the setter private and only expose the possibility to get the number of players.